notebooks/500_use_cases/501_dobot/501b_inference_with_a_robotic_arm.ipynb

```diff
@@ -1,14 +1,16 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.        metadata_path = weights_path / "openvino" / "meta_data.json"
</pre><p>Typo: metadata instead of meta_data</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/anomalib/pull/1084/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>